src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs

```diff
@@ -437,4 +437,218 @@ internal enum WebRTCDisplayType
 
         Evas,
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// Specifies the type of WebRTC statistics.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    [Flags]
+    public enum WebRTCStatisticsCategory
+    {
+        /// <summary>
+        /// Codec.
+        /// </summary>
+        Codec = 1,
```

Comment:
Could we set value with 0x0001 style? this could be used with bitwise operation.

Reply:
It can be used same way because it have Flag attribute.

Reply:
I'll change this hex style to keep consistency with existing internal enum.